README.md

```diff
@@ -26,6 +26,72 @@ Run prettier to lint and write - this is also done automatically in a pre-commit
 yarn lint
 ```
 
+## Deploying to remote networks
+Requirement: `gcloud` must be set up and user account must have access staked-celo-alfajores and staked-celo-staging on GCP. 
+Next, ensure the environment variables have been decrypted, using `yarn keys:decrypt`.
+Then use the following commands to deploy depending on the desired target environment.
+For example, the below command will deploy to the Alfajores network, using the decrypted private key, the default Alfajores rpc url and the other variables in `.env.alfajores`. 
+
+Alfajores : 
+```
+yarn deploy --network alfajores
+```
+
+You may immediately verify the deployment, with the following commands.
+
+Alfajores : 
+```
+yarn verify --network alfajores
+```
+
+You may desire to deploy using an unlocked account in a private node. In that case, you can use the following commands :
```

Comment:
Perhaps, to be clearer, adding this section under a new heading `## Deploying to local CELO node` that include `### Steps to Run a light node:` && the deploy command.

That way we can have the section where we explained how to run the node and what command to use when deploying using that node.